src/Sulu/Bundle/MediaBundle/Resources/js/containers/Form/fields/MediaVersionUpload.js

```diff
@@ -2,8 +2,8 @@
 import React from 'react';
 import {ResourceStore} from 'sulu-admin-bundle/stores';
 import type {FieldTypeProps} from 'sulu-admin-bundle/types';
+import ResourceFormStore from 'sulu-admin-bundle/containers/Form/stores/ResourceFormStore';
```

Comment:
Use `sulu-admin-bundle/stores` to import that file

Reply:
looks like `ResourceFormStore` is not exported in `sulu-admin-bundle/stores`. but i will use `sulu-admin-bundle/containers`